packages/admin-ui/src/__mock-backend__/wifi.ts

```diff
@@ -9,7 +9,10 @@ export const wifi: Pick<Routes, "wifiCredentialsGet" | "wifiReportGet"> = {
       containerState: "exited",
       info: "Wifi service exited due to an internal error",
       report: {
-        lastLog: `[Warning] No interface found after 60s, stopping gracefully`,
+        lastLog: "[Warning] No interface found after 60s, stopping gracefully".replace(
+          /\[.*?\]/g,
+          ""
+        ),
```

Comment:
What is this regex doing? Since this is a static text could you just use the result?

Reply:
Oh I see. Hmm, is this user-friendly enough? Just removing this tag and showing the message on screen is good? What is the user expected to do when he/she sees
```
No interface found after 60s, stopping gracefully
```
?

Reply:
I tried to make it as friendly as possible and be as consistent as possible. To do that I return last error log when wifi container was exited/dead and render it as markdown code

I cannot prevent how is going to be every single error log from the wifi container, also hardcode them may be risky due to it can get unupdated soon.

Im open to any other suggestion